superset-frontend/src/dashboard/components/DashboardBuilder/DashboardBuilder.tsx

```diff
@@ -182,8 +182,9 @@ const DashboardBuilder: FC<DashboardBuilderProps> = () => {
       ? dashboardLayout[rootChildId]
       : undefined;
   const isStandalone = getUrlParam(URL_PARAMS.standalone);
```

Comment:
This was a bit confusing, the name `isStandalone` suggests that this is a boolean. We should rename this to `standaloneMode` or something more clear.